tests/client/core/actions/test_index.js

```diff
@@ -28,6 +30,35 @@ describe('core actions setClientApp', () => {
   });
 });
 
+describe('core actions setUserAgent', () => {
+  it('creates the SET_USER_AGENT action', () => {
+    const userAgent = userAgents.chrome[0];
+
+    assert.deepEqual(actions.setUserAgent(userAgent), {
+      type: 'SET_USER_AGENT',
+      payload: { userAgent },
+    });
+  });
+
+  it('allow an empty userAgent value', () => {
```

Comment:
I think this test is no longer useful here. It should move to the reducer since that's where the parsing logic happens.